std_msgs/Header header

float32[3] pos
float32[3] vel

float32[3] error_pos     
float32[3] error_vel    


float32[3] u_l
float32[3] u_d
float32[3] u_total

float32[3] thrust_sp



